README.md

```diff
@@ -101,11 +101,11 @@ it is not maintained in git - and thus is not pulled down from github. Contact
 one of the project members to get the contents of that file (for example via
 private message in Slack, or general message in the project's Slack channel).
 
-### Step 4: Update the database
+### Step 4: Set up the database
 ```shell
 $ bundle exec rake shf:db_recreate
 ```
-The rake task `db_recreate` creates the development DB, creates the application
+The rake task `shf:db_recreate` recreates the development DB, creates the application
```

Comment:
Minor text suggestion:
change
 "The rake task  `shf:db_recreate` recreates the development DB..."
to
"The rake task  `shf:db_recreate` **creates or** recreates the development **and test** DBs..."
(I emphasized the changed; I don't mean that formatting to be included.)

If the db(s) don't exist, they will be created.  This is what people setting up the system for the first time want to do.  I think if we don't mention that, novices might wonder how to accomplish that.

Also, this _will_ work on both the `development` and `test` dbs/environments.